Mathlib/Topology/Algebra/Subgroup.lean

```diff
@@ -1,11 +1,12 @@
 /-
 Copyright (c) 2019 Johan Commelin. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johan Commelin
+Authors: Johan Commelin, Nailin Guan
```

Comment:
The module docstring for this file needs an update, given the changes (and indeed the change of purpose of this file plus its renaming). But I would suggest that instead the material for closed subgroups goes into a new file `ClosedSubgroup` and this file is not renamed but just takes on the open normal subgroup material (and then the module docstring is updated to reflect this).

Reply:
split as suggested